modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/type/runtime/MethodWrapper.java

```diff
@@ -143,6 +145,12 @@ public <A extends Annotation> Optional<AnnotationValueFetcher<A>> getValueFromAn
         : empty();
   }
 
+  @Override
+  public List<Type> getAnnotations() {
+    return Arrays.stream(method.getAnnotations()).map(ann -> new TypeWrapper(ann.annotationType(), typeLoader))
```

Comment:
Can we add some unit tests for these new methods ?

Reply:
static import